Gemfile

```diff
@@ -78,6 +78,7 @@ gem 'discourse_image_optim', require: 'image_optim'
 gem 'multi_json'
 gem 'mustache'
 gem 'nokogiri'
+gem 'css_parser'
```

Comment:
Did we have a discussion about this gem anywhere that I might have missed? We are very stingy about adding new things and must do due-diligence to make sure they meet our standards.

Reply:
Nope, and I'd be happy to use another solution to do the same. Was hoping nokogiri could do this, but it doesn't.

Reply:
Okay I've created a document on our developer forum about the dependency. If we add the dependency it should be `require: false` and only required when we intend to use it, since it's probably a rarely used code path.